packages/pn-commons/src/utils/genericFunctions.utility.tsx

```diff
@@ -1,3 +1,7 @@
 export function getValidValue(a: string | number | undefined, b?: string | number): any {
   return a || (b ? b : '');
 }
+
+export function getDefaultDate(date1: Date, date2: Date, date3: string): any {
```

Comment:
@NicolaGiornetta dai nomi di questi parametri non mi è chiara la relazione e cosa dovrebbe fare questo metodo. Cambiamo i nomi dei parametri e aggiungiamo un jsdoc.

Reply:
@NicolaGiornetta come da confronto andiamo ad eliminare questo check. è sufficiente settare nei filtri il valore contenuto in filters